common/softwarecontainer-common.cpp

```diff
@@ -157,10 +157,6 @@ SignalConnectionsHandler::~SignalConnectionsHandler()
     }
 }
 
-bool operator&& (ReturnCode lhs, ReturnCode rhs) {
-    return lhs == rhs;
-}
```

Comment:
This operator makes no sense.
```cpp
ReturnCode::FAILURE && ReturnCode::FAILURE
```
does **not** equal
```cpp
ReturnCode::FAILURE == ReturnCode::FAILURE
```
Use isError() or isSuccess() intead